src/quo2/screens/main.cljs

```diff
@@ -124,12 +132,20 @@
                         :padding-horizontal 16
                         :background-color   (:ui-background @colors/theme)}
         [theme-switcher]
+        [quo2-text/text {:size :heading-1} "Preview Quo2 Components"]
+
         [rn/view
-         (for [{:keys [name]} screens]
-           ^{:key name}
-           [rn/touchable-opacity {:on-press #(re-frame/dispatch [:navigate-to name])}
-            [rn/view {:style {:padding-vertical 8}}
-             [quo/text (str "Preview " name)]]])]])]))
+         (map (fn [category]
+                [rn/view  {:style {:margin-vertical 8}}
+                 [quo2-text/text
+                  {:weight :semi-bold
+                   :size :heading-2}
+                  (clojure.core/name (key category))]
+                 (for [{:keys [name]} (val category)]
+                   ^{:key name}
+                   [rn/touchable-opacity {:on-press #(re-frame/dispatch [:navigate-to name])}
```

Comment:
probably better to use quo2 button here, because now it will be difficult to press, small and too close and it doesn't look like a button

Reply:
cool, adjusted to use a button instead.